build.sbt

```diff
@@ -645,6 +645,7 @@ def osgiTestProject(p: Project, framework: ModuleID) = p
       )
     },
     Keys.test in Test := (Keys.test in Test).dependsOn(packageBin in Compile).value,
+    Keys.testOnly in Test := (Keys.test in Test).dependsOn(packageBin in Compile).value,
```

Comment:
I don't even understand this, sorry. Why isn't it `Keys.testOnly` on RHS? This is why for years now I demur from learning sbt internals.

Reply:
Pushed a fix. I tested it by ensuring that I could add a `???` to the implementation of reflection and witness the failure from:

```
> osgiTestEclipse/testOnly
--                                                         tools.test.osgi.BasicTest                                  tools.test.osgi.libonly.BasicLibraryTest
tools.test.osgi.reflection.basic.BasicReflectionTest       tools.test.osgi.reflection.toolbox.ReflectionToolBoxTest
> osgiTestEclipse/testOnly
--                                                         tools.test.osgi.BasicTest                                  tools.test.osgi.libonly.BasicLibraryTest
tools.test.osgi.reflection.basic.BasicReflectionTest       tools.test.osgi.reflection.toolbox.ReflectionToolBoxTest
> osgiTestEclipse/testOnly tools.test.osgi.reflection.basic.BasicReflectionTest
...
[error] Test tools.test.osgi.reflection.basic.BasicReflectionTest.basicMirrorThroughOsgi failed: scala.NotImplementedError: an implementation is missing, took 0.886 sec
[error]     at scala.Predef$.$qmark$qmark$qmark(Predef.scala:341)
```